### ISSUE TITLE
Infinite multiplied by a integer raises an error

### ISSUE THREAD

**Issue by kikocorreoso** [brython-dev/brython#352]
The following:
```python
a = float("inf")
print(a*1)
```
Should return (CPython3)
```python
-inf
```
It returns
```javascript
Traceback (most recent call last):
  module editor line 100
    traceback.print_exc(file=sys.stderr)
  module $exec_213 line 2
    a*1
ValueError: argument of long_int must be a string, not float
<completed in  11.00 ms>
```

**Comment by kikocorreoso** [comment]
It seems the error is related with operations defined in *py_utils.js*.

**Comment by olemis** [comment]
@kikocorreoso I'm getting `inf` rather than `-inf`

```
>>> float("inf") * 1
inf
```



On 12/16/15, kikocorreoso <notifications@github.com> wrote:
> Reopened #352.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/brython-dev/brython/issues/352#event-494112524


-- 
Regards,

Olemis - @olemislc

Apache™ Bloodhound contributor
http://issues.apache.org/bloodhound
http://blood-hound.net

Brython committer
http://brython.info
http://github.com/brython-dev/brython

Blog ES: http://simelo-es.blogspot.com/
Blog EN: http://simelo-en.blogspot.com/

Featured article:

**Comment by kikocorreoso** [comment]
My mistake.

I wanted to post your example. In Cpython it works as expected but in Brython raises an error as the operators defined in *py_utils.js* doesn't take into account this cases...

For example:

```python
a = float("inf")
print(a * 1.)
```
works partially, returns `Infinity` but as a js *inf* instead of a python `inf`.

But if you multiply with an integer it results in an internal js error.
```python
a = float("inf")
print(a * 1)
```

**Comment by PierreQuentel** [comment]
Hello,

I was working on the tests in Lib/test/test_float.py, using the new tool at /tests/unittests/index.html (did you try it ?), so I fixed this bug with the others in commit 53508eb384612ee896bbdcfc743b3081ff091cf7

**Comment by kikocorreoso** [comment]
Great it is fixed!!!

I'm working in the port of some new stdlibs to Brython and I'm finding some problems/bugs with the data model...

Yes. I saw the unittest tool. The problem with it is that sometimes is so heavy that the browser is unresponsive.

Thanks @PierreQuentel
